snorkel/labeling/utils.py

```diff
@@ -27,3 +28,45 @@ def filter_unlabeled_dataframe(
     """
     mask = (L != -1).any(axis=1)
     return X.iloc[mask], y[mask]
+
+
+def create_cv_data(
```

Comment:
would recommend keeping in wrapper so that there's a single, canonical import statement (i.e. everyone thinks to always import this when importing the wrapper

Reply:
a little torn on what to name this...we aren't using CV but we are sticking it in a CV routine...maybe just clarify in doc string.